src/ui/templates/results/verticalresultscount.hbs

```diff
@@ -1,19 +1,21 @@
 {{#unless (eq total 0)}}
-  <div class="yxt-VerticalResultsCount"
-    aria-label="{{translate phrase='[[start]] through [[end]] of [[resultsCount]]' start=pageStart end=pageEnd resultsCount=total }}">
-    <div aria-hidden="true">
-      {{translate
-        phrase=
+  {{#unless (and isNoResults hiddenForNoResults)}}
```

Comment:
Can we combine the two `unless` statements ?
```
{{#unless (or (eq total 0) (and isNoResults hiddenForNoResults)) }}
```
Alternatively, in the JS, you could just make a variable called `shouldDisplayCount` that encapsulates this compound check.

Reply:
sure